cdec/installation-manager-core/src/main/java/com/codenvy/im/managers/helper/NodeManagerHelperCodenvy4Impl.java

```diff
@@ -288,6 +306,39 @@ public void validateLicense() throws IOException {
         }
     }
 
+    private boolean isCodenvy4Installed() throws IOException {
+        Optional<Version> version = createArtifact(CDECArtifact.NAME).getInstalledVersion();
```

Comment:
isCodenvy4Installed is really bad name.
Soon we will release 5.x and it will looks silly